guides/cancelOrders.md

```diff
@@ -49,25 +32,21 @@ const receipt = await tx.wait();
 This function can be used to cancel all the sender's bids or all the sender's asks, or both in a single call. The following example showcases all the possible combinations.
 
 ```ts
-import { LooksRare, ChainId } from "@looksrare/sdk-v2";
+import { HypercertExchangeClient, ChainId } from "@hypercerts-org/marketplace-sdk";
 
-const lr = new LooksRare(ChainId.MAINNET, provider, signer);
+const hypercertExchangeClient = new HypercertExchangeClient(ChainId.OPTIMISM, provider, signer);
 
 // Cancel all bids
-const tx = await lr.cancelAllOrders(true, false).call();
+const tx = await hypercertExchangeClient.cancelAllOrders(true, false).call();
 const receipt = await tx.wait();
 
 // Cancel all asks
-const tx = await lr.cancelAllOrders(false, true).call();
+const tx = await hypercertExchangeClient.cancelAllOrders(false, true).call();
 const receipt = await tx.wait();
 
 // Cancel all bids and asks
-const tx = await lr.cancelAllOrders(true, true).call();
+const tx = await hypercertExchangeClient.cancelAllOrders(true, true).call();
 const receipt = await tx.wait();
 ```
 
-For more details on our triple-nonce system, see the related documentation: [Triple-nonce system](https://docs.looksrare.org/developers/protocol/triple-nonce-system-v2)
-
-## Need help?
-
-You can reach out to the LooksRare team via our Developers Discord: [https://discord.gg/LooksRareDevelopers](https://discord.gg/LooksRareDevelopers)
+For more details on the triple-nonce system, see the related documentation: [Triple-nonce system](https://docs.looksrare.org/developers/protocol/triple-nonce-system-v2)
```

Comment:
Let's host that page ourselves

Reply:
Write our own? What if their page goes down?

Reply:
Added the docs.

Reply:
@bitbeckers added it back after discussion with @holkexyz . Kept the missing subset nonce docs in there as well commented out, so we can show that once we've implemented the API part of it.